UNIXFSv1.md

```diff
@@ -0,0 +1,487 @@
+# ![](https://img.shields.io/badge/status-wip-orange.svg?style=flat-square) UnixFS  <!-- omit in toc -->
+
+**Author(s)**:
+- NA
+
+* * *
+
+**Abstract**
+
+UnixFS is a [protocol-buffers](https://developers.google.com/protocol-buffers/) based format for describing files, directories, and symlinks as merkle-dags in IPFS.
+
+Draft work and discussion on a specification for the upcoming version 2 of the UnixFS format is happening in the [`ipfs/unixfs-v2` repo](https://github.com/ipfs/unixfs-v2). Please see the issues there for discussion and PRs for drafts.
+
+## Table of Contents <!-- omit in toc -->
+
+- [Implementations](#implementations)
+- [Data Format](#data-format)
+- [Metadata](#metadata)
+	- [Deduplication and inlining](#deduplication-and-inlining)
+- [Importing](#importing)
+	- [Chunking](#chunking)
+	- [Layout](#layout)
+- [Exporting](#exporting)
+- [Design decision rationale](#design-decision-rationale)
+	- [Metadata](#metadata-1)
+		- [Separate Metadata node](#separate-metadata-node)
+		- [Metadata in the directory](#metadata-in-the-directory)
+		- [Metadata in the file](#metadata-in-the-file)
+		- [Side trees](#side-trees)
+		- [Side database](#side-database)
+
+## Implementations
+
+- JavaScript
+  - Data Formats - [unixfs](https://github.com/ipfs/js-ipfs-unixfs)
+  - Importer - [unixfs-importer](https://github.com/ipfs/js-ipfs-unixfs-importer)
+  - Exporter - [unixfs-exporter](https://github.com/ipfs/js-ipfs-unixfs-exporter)
+- Go
+  - [`ipfs/go-ipfs/unixfs`](https://github.com/ipfs/go-ipfs/tree/b3faaad1310bcc32dc3dd24e1919e9edf51edba8/unixfs)
+  - Protocol Buffer Definitions - [`ipfs/go-ipfs/unixfs/pb`](https://github.com/ipfs/go-ipfs/blob/b3faaad1310bcc32dc3dd24e1919e9edf51edba8/unixfs/pb/unixfs.proto)
+
+## Data Format
```

Comment:
> note that both can be interpreted as UnixFS, and `raw` can be used for filles that are smaller than 1MiB (or whatever effective limit exists in the ecosystem) and leaves.

This is exactly what I tried to do in `Raw blocks` section and subsections, are you proposing to do this because I failed at explaining this ?

Reply:
I'll change the order, it's easier to follow.